Makefile

```diff
@@ -76,16 +76,16 @@ install-ci:
 	# curl -sSL -o helm.tar.gz https://get.helm.sh/helm-v${HELM_VERSION}-${HELM_PLATFORM}-amd64.tar.gz
 	# tar -xvzf helm.tar.gz
 	# mv ${HELM_PLATFORM}-amd64/helm $(GOPATH)/bin/
-	go install github.com/google/addlicense
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.44.0
```

Comment:
We should not need this. These dependencies are pulled based on the version in the go.mod file. Updating the version on go mod should suffice

Reply:
This is what I removed since we don't need these deps at runtime and just need to install the lint command. See the deleted `tools.go` file

Reply:
otherwise `go.mod` gets polluted by all build time deps as well.